docs/ams-schema-spec.html

```diff
@@ -3715,7 +3715,7 @@ <h2 class="heading settled" data-level="7" id="attr-types"><span class="secno">7
      <tr>
       <td><dfn class="dfn-paneled idl-code" data-dfn-type="typedef" data-export data-lt="id|id-type" id="typedefdef-id"><code class="highlight">identifier</code></dfn>
       <td> (string | integer)
-      <td> Een unieke identifier in de vorm van een integer of een alphanumerieke reeks, startend met een lowercase letter.
+      <td> Een unieke identifier in de vorm van een letter reeks, startend met een lowercase letter eventueel gevolgd door een cijferreeks voorafgegaan door een underscore '_'.
```

Comment:
_in de vorm van een letter reeks_ `->` _in de vorm van één of meer letters_

If _reeks_ however is what is actually meant, then please see this [comment](https://github.com/Amsterdam/amsterdam-schema/pull/323/files#r756611709)